easybuild/easyconfigs/o/Oases/Oases-0.2.08-intel-2017b-kmer_101.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+#
+##
+
+easyblock = "MakeCp"
+
+name = 'Oases'
+version = '0.2.08'
+versionsuffix = '-kmer_101'
+
+homepage = 'http://www.ebi.ac.uk/~zerbino/oases/'
+description = """Oases is a de novo transcriptome assembler designed to produce transcripts from
+ short read sequencing technologies, such as Illumina, SOLiD, or 454 in the absence of any genomic assembly."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+velvetver = '1.2.10'
+
+source_urls = [
+    'http://www.ebi.ac.uk/~zerbino/%(namelower)s',
+    'http://www.ebi.ac.uk/~zerbino/velvet',
```

Comment:
@smoors  These source URLs don't work anymore. Is the code hosted elsewhere now?

Reply:
@verdurin For me they still work, could you please try again?

Reply:
Still works for me too, still provides exact same source tarballs (SHA256 checksum is identical for fresh downloads).

So +1 on trying again @verdurin ;)